.github/settings.yml

```diff
@@ -0,0 +1,12 @@
+# The settings are inherited from common repository repo-settings (same file name and location)
+# You can append settings that are deeply-merged with the inherited settings.
+#
+# When you have installed the GitHub App "repo-settings" in this repository,
+# any change of this settings.yml file is detected by the GitHub App and
+# the settings of this repository are updated immediately.
+#
+_extends: repo-settings
+
+repository:
+  # The name of the repository. Changing this will rename the repository
+  name: terraform-ibm-cos
```

Comment:
description would be aligned with what is currently used internally = "Creates generic COS instance and bucket with a variety of options "